website/docs/reference/resource-configs/alias.md

```diff
@@ -25,9 +62,8 @@ To learn more about changing the way that dbt generates a relation's `identifier
 ## Usage
```

Comment:
Do you think we still need this section based on the examples you've provided above? I think we could remove it cos it's largely repeating itself?

Reply:
i was going to do exactly that but then i wondered how many ppl might have the #usage anchor link? and since we can't redirect anchor links to another link, i stayed conservative 😭